### PR TITLE
install scikit-learn in alpine-analytics since it's slow

### PR DESCRIPTION
**Pull request by erikbern**
It takes a few minutes to install, kind of annoying

### CODE REVIEW

File: Dockerfile.analytics [hunk 1]
```diff
@@ -18,7 +18,8 @@ RUN pip3 install matplotlib==2.0.2 \
     psycopg2==2.7.2 \
     seaborn==0.8 \
     pandas==0.22.0 \
-    https://github.com/better/alpine-tensorflow/releases/download/alpine3.7-tensorflow1.7.0/tensorflow-1.7.0-cp36-cp36m-linux_x86_64.whl
+    https://github.com/better/alpine-tensorflow/releases/download/alpine3.7-tensorflow1.7.0/tensorflow-1.7.0-cp36-cp36m-linux_x86_64.whl \
+    scikit-learn==0.19.1
```
Comment (imsky) [inline Dockerfile.analytics]: Maybe more clean to add official packages before unofficial ones

**Comment by imsky** [comment]
:+1:

File: Dockerfile.analytics (hunk 1 above)
Comment (erikbern) [reply Dockerfile.analytics]: actually my preference is to put them in alphabetical order. i can do that.

**Comment by erikbern** [comment]
Had to break up the pip command for some reason. scikit-learn depends on scipy but doesn't declare its dependency annoyingly

**Comment by erikbern** [comment]
Thanks @imsky

**Pull request merged**